Sign_Up_Demo/Controllers/ViewController.swift

```diff
@@ -56,13 +57,12 @@ class ViewController: UIViewController {
         
     }
     
-    private func showAlert(message: String) {
+    private func showAlertWithMsg(message: String) {
```

Comment:
Redundancy in method name.
either it should be showAlertWith(message: String) 
or 
showAlertWithMessage(_ message: String)

also never use short forms like Msg in this case. Always prefer to use full names.

Reply:
And this method should be placed somewhere globally. May be in UiViewController extension so it would be available in all ViewControllers or may be a static method in some class with name Utils, Helpers, Globals etc

Reply:
Always try to reuse the code